lib/DxilPIXPasses/DxilAnnotateWithVirtualRegister.cpp

```diff
@@ -123,8 +123,8 @@ bool DxilAnnotateWithVirtualRegister::runOnModule(llvm::Module &M) {
   }
   unsigned int Major = 0;
   unsigned int Minor = 0;
-  m_DM->GetDxilVersion(Major, Minor);
-  if (Major < 6 || (Major == 6 && Minor <= 4)) {
+  m_DM->GetValidatorVersion(Major, Minor);
```

Comment:
Wha's the original reason for upgrading the validator version to 1.4 specifically?

Reply:
I wish I could ask jpporto that question... this pass does nothing but add some metadata, so maybe the metadata format changed in 1.4? I also wonder if it's here on behalf of the two other passes that are typically run with this one (value-to-delcare and shader debug instrumentation).
I guess I could hack this to assert 1.3 and see what breaks... I'd be pretty nervous about removing this though, even if nothing I try actually breaks.